addon/ng2/blueprints/route/index.js

```diff
@@ -1,16 +1,90 @@
-var stringUtils = require('ember-cli/lib/utilities/string');
+var fs = require('fs-extra');
+var path = require('path');
+var chalk = require('chalk');
 
+var imports, routeDefinitions;
+
+var simpleStringify = function (object) {
+  var simpleObject = {};
+  for (var prop in object) {
+    if (!object.hasOwnProperty(prop)) {
+      continue;
+    }
+    if (typeof(object[prop]) == 'object') {
+      continue;
+    }
+    if (typeof(object[prop]) == 'function') {
+      continue;
+    }
+    simpleObject[prop] = object[prop];
+  }
+  return JSON.stringify(simpleObject); // returns cleaned up JSON
+};
 module.exports = {
-  description: ''
-
-  //locals: function(options) {
-  //   // Return custom template variables here.
-  //   return {
-  //
-  //   };
-  //}
-
-  // afterInstall: function(options) {
-  //   // Perform extra work here.
-  // }
+  description: 'Generates a route and a template.',
+
+  // TODO these options are not being used yet
+  availableOptions: [{
+    name: 'skip-router-generation',
+    type: Boolean,
+    default: false,
+    aliases: ['srg']
+  }, {
+    name: 'default',
```

Comment:
I was wondering why I had this `default` property here in the original PR.... it's for the `useAsDefault` functionality mentioned above.